config/upstreams.go

```diff
@@ -1,29 +1,42 @@
 package config
 
 import (
+	"github.com/0xERR0R/blocky/log"
 	"github.com/sirupsen/logrus"
 )
 
 const UpstreamDefaultCfgName = "default"
 
 // Upstreams upstream servers configuration
 type Upstreams struct {
-	Timeout     Duration         `yaml:"timeout" default:"2s"`
-	Groups      UpstreamGroups   `yaml:"groups"`
-	Strategy    UpstreamStrategy `yaml:"strategy" default:"parallel_best"`
-	StartVerify bool             `yaml:"startVerify" default:"false"`
```

Comment:
Please document this change in docs/config.yaml and docs/configuration.md

Reply:
Oops, thanks for the reminder!